modules/document_repository/php/NDB_Menu_Filter_document_repository.class.inc

```diff
@@ -77,9 +77,11 @@ class NDB_Menu_Filter_Document_Repository extends NDB_Menu_Filter
                           'v.Date_uploaded',
                          );
 
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = isset($_GET['openAccordion']) ?
```

Comment:
But this will still compile in php 7. So, it can still be merged now and will work.